### PR TITLE
Force flush

### PR DESCRIPTION
**Pull request by msporny**

### CODE REVIEW

File: lib/LedgerNodeOperations.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ module.exports = class LedgerNodeOperations {
    *
    * @return {Promise} resolves once the operation completes.
    */
-  async add({operation}) {
+  async add({operation, forceFlush}) {
```
Comment (dlongley) [inline lib/LedgerNodeOperations.js:28]:
```suggestion
  async add({operation, options}) {
```

File: lib/LedgerNodeOperations.js [hunk 2]
```diff
@@ -38,7 +38,7 @@ module.exports = class LedgerNodeOperations {
       throw result.error;
     }
     const meta = {basisBlockHeight};
-    return consensus.operations.add({ledgerNode, meta, operation});
+    return consensus.operations.add({ledgerNode, meta, operation, forceFlush});
```
Comment (dlongley) [inline lib/LedgerNodeOperations.js:41]:
```suggestion
    return consensus.operations.add({ledgerNode, meta, operation, options});
```

**Review by dlongley — CHANGES_REQUESTED** [review]
This was mentioned in another PR, but `options` should be passed through here at this level, not any particularly specific option.

**Comment by msporny** [comment]
Merging into get-ledger-node-record and fixing build errors up there (with old dependencies).

**Pull request merged**